stdlib/public/TensorFlow/ArrayOps.swift

```diff
@@ -196,4 +196,43 @@ public extension Raw {
     }
     return out
   }
+
+  /// Unpacks a tensor into `numSplit` tensors along one dimension.
```

Comment:
Look like there's no parameter called `numSplit`. Is it the one called `num`?

```suggestion
  /// Unpacks a tensor into `num` tensors along one dimension.
```